cmd/scollector/collectors/program.go

```diff
@@ -115,7 +116,12 @@ func (c *ProgramCollector) Init() {
 var setupExternalCommand = func(cmd *exec.Cmd) {}
 
 func (c *ProgramCollector) runProgram(dpchan chan<- *opentsdb.DataPoint) (progError error) {
-	cmd := exec.Command(c.Path)
+	var cmd *exec.Cmd
+	if runtime.GOOS == "windows" && filepath.Ext(c.Path) == ".ps1" {
+		cmd = exec.Command("powershell", "-NoProfile", "-ExecutionPolicy", "Bypass", "-File", c.Path)
```

Comment:
On second thought... ENV vars don't work well with Windows Services (usually require a reboot to change). How do you feel about adding a PSExecutionPolicy string to https://github.com/bosun-monitor/bosun/blob/master/cmd/scollector/conf/conf.go#L41 and only including it in the command line args if the value is provided? I think that would be the best path forward, but @captncraig might also have an opinion.